packages/commonwealth/client/scripts/views/Sublayout.tsx

```diff
@@ -81,11 +78,15 @@ const Sublayout = ({
         <div className="sidebar-and-body-container">
           <Sidebar isInsideCommunity={hasCommunitySidebar} />
           <div
-            className={clsx('body-and-sticky-headers-container', {
-              'menu-visible': menuVisible,
-              'menu-hidden': !menuVisible,
-              resizing,
-            })}
+            className={clsx(
+              'body-and-sticky-headers-container',
+              menuVisible ? 'menu-visible' : 'menu-hidden',
+              (menuName === 'exploreCommunities' ||
+                menuName === 'createContent' ||
+                hasCommunitySidebar) &&
+                'quick-switcher-visible',
+              resizing
+            )}
```

Comment:
clsx is great tool to write dynamic classes in readable style, not sure why you don't want to take advantage of that?

```
          <div
            className={clsx(
              'body-and-sticky-headers-container',
              {
                'menu-visible': menuVisible,
                'menu-hidden': !menuVisible,
                'quick-switcher-visible':
                  menuName === 'exploreCommunities' ||
                  menuName === 'createContent' ||
                  hasCommunitySidebar,
              },
              resizing
            )}
          >

```

you provide the class and then provide the condition. If condition is truthy, the class is added. Otherwise, it is not. Very simple and readable